### PR TITLE
post rc cleanup

### PR DESCRIPTION
**Pull request by josephcopenhaver**

### CODE REVIEW

File: magefiles/magefile.go [hunk 1]
```diff
@@ -551,19 +550,18 @@ func Shell(ctx context.Context) error {
...
 }
 
 func Logs(ctx context.Context) error {
 	mg.Deps(vars)
 
-	argsb := CmdB().Fields("logs -f")
+	args := strings.Fields("logs -f")
 	if s := os.Getenv("SERVICES"); s != "" {
-		argsb.Fields(s)
+		args = append(strings.Fields(strings.TrimSpace(s)))
```
Comment (jpcope) [inline magefiles/magefile.go:561]: oi, args is missing on append func and fields should be varg'd

Comment (josephcopenhaver) [reply magefiles/magefile.go:561]: fixed in https://github.com/josephcopenhaver/melody-bot/pull/25

**Comment by jpcope** [comment]
lint config should have caught this. The single argument variant is a nop at compile time and is almost never what the author intends.

This oversight has been discussed previously and there is a comment that staticcheck has a lint for this. https://github.com/golang/go/issues/30040

from https://golangci-lint.run/usage/linters/

`staticcheck
It's a set of rules from staticcheck. It's not the same thing as the staticcheck binary. The author of staticcheck doesn't support or approve the use of staticcheck as a library inside golangci-lint.`

**Comment by jpcope** [comment]
aha, likely build-tags argument of golangci-lint should be altered.

https://github.com/josephcopenhaver/melody-bot/blob/b018609c9fc289cb92a7b9444b69a14c21c758f5/.golangci.yaml#L10

test should be unit and mage should be added to this list.

**Pull request merged**